tests/user_me_query.spec.js

```diff
@@ -0,0 +1,169 @@
+const axios = require('axios');
+const { URL } = require('../constants');
+const getToken = require('./functions/getToken');
+
+let token;
+
+beforeAll(async () => {
+  token = await getToken();
+});
+
+describe('user resolvers', () => {
+  test('Get Logged In User details - me Query', async () => {
+    const response = await axios.post(
+      URL,
+      {
+        query: `
+      {
+        me {
+          _id
+          firstName
+          lastName
+          email
+          userType
+          appLanguageCode
+          createdOrganizations {
+            _id
+            name
+            description
+            isPublic
+            visibleInSearch
+            apiUrl
+            creator {
+              _id
+              firstName
+            }
+            members {
+              _id
+              firstName
+            }
+            admins {
+              _id
+              firstName
+            }
+            membershipRequests {
+              _id
+            }
+            blockedUsers{
+              _id
+            }
+          }
+        }
+      }
+      `,
+      },
+      {
+        headers: {
+          Authorization: `Bearer ${token}`,
+        },
+      }
+    );
+
+    const { data } = response;
+
+    expect(data.data.me).toEqual(
```

Comment:
Can you please create a test code where you first create a user (using some random variable for the cases like email) and then check if the results expected are coming? Currently, all cases are just checking if the result is string

Reply:
I think that we don't need to signup again. Instead, we can get userId from `getUserId` function , and values used for creating a user in `getToken` function. And verify them in `me` Query Test.

Reply:
ok